CONTRIBUTING.md

```diff
@@ -6,10 +6,11 @@ The CodeSandbox client is currently divided in to 5 parts. We use `lerna` to sha
 
 - `app`: The editor, the search, profile page, the embed and the sandbox.
 - `homepage`: The Gatsby website of the homepage.
-- `codesandbox-api`: The npm package that's responsible for communication between the sandbox and the editor.
 - `common`: All common parts between these packages, reusable JS.
+- `codesandbox-api`: The npm package that's responsible for communication between the sandbox and the editor.
+- `codesandbox-browserfs`: An in-browser file system that emulates the Node JS file system API and supports storing and retrieving files from various backends. Forked from [https://github.com/jvilk/BrowserFS](https://github.com/jvilk/BrowserFS), with an additional [CodeSandbox backend](https://github.com/CompuIves/codesandbox-client/blob/master/standalone-packages/codesandbox-browserfs/src/backend/CodeSandboxFS.ts).
 
-This version of CodeSandbox is using the production server as source of truth, this is specified by the environment variable `LOCAL_SERVER`. It's not yet possible to sign in using this version, I haven't figured out how to handle this yet.
+This version of CodeSandbox is using the production server as source of truth, this is specified by the environment variable `LOCAL_SERVER`. If you're working on a feature that needs you to be logged in, you can login on [https://codesandbox.io/](https://codesandbox.io/) and copy the contents of the `jwt` local storage key over to your development environment on [http://localhost:3000/](http://localhost:3000/). **Be very careful with how you handle the token**, as anyone who knows it can login as you and have read/write access to all your CodeSandbox content!
```

Comment:
I'm not sure if we could recommend copying `jwt` token as an official way, since it feels VERY insecure. It could be safe though, I mean the JWT is sent insecure to localhost, but it's localhost, and then sent over https. I'm not sure how to feel about this, what do you think?

Reply:
I had the same concerns and came to the same conclusions. It's still opt-in, would probably only be used by people that work on features that require to be logged-in, and there's also the warning.

Reply:
Agreed, okay let's keep it then. Maybe we can change the dev environment in the near future to work on codesandbox.stream api, feels much safer. Then we should probably add a header in the proxy so nginx knows not to redirect.